src/coreclr-debug.ts

```diff
@@ -24,6 +26,8 @@ export function installCoreClrDebug(context: vscode.ExtensionContext) {
         return;
     }
     
+    initializeTelemetry(context);

```

Comment:
Might want to move this to line 36 for now so that we do as little as possible when .NET CLI isn't installed.